packages/flutter/lib/src/scheduler/binding.dart

```diff
@@ -720,6 +720,8 @@ mixin SchedulerBinding on BindingBase {
       scheduleFrame();
   }
 
+  /// Ensures callbacks for [window.onBeginFrame] and [window.onDrawFrame]
```

Comment:
Dartdoc cannot resolve these references: https://cirrus-ci.com/task/6118617987153920?command=main#L135

You probably have to capitalize the "w" in Window to resolve.

Reply:
Hmmm, I'm not sure the capital is working either.  Maybe these don't need to be actual element references?

Something like:

     /// Ensures callbacks for `window.onBeginFrame` and `window.onDrawFrame`

Reply:
(I'll give this a whirl.)

Reply:
capital letter should work...